src/components/audio-trimmer/audio-selector.css

```diff
@@ -0,0 +1,119 @@
+@import "../../css/colors.css";
```

Comment:
We should probably see if we can share the styles between this and the existing audio trimmer, then use classes for the color selection

Reply:
did it